packs/rabbitmq/config.yaml

```diff
@@ -0,0 +1,6 @@
+sensor_config:
+  poll_time: 5
+  queues:
```

Comment:
I feel that way translates pretty poorly to the readme in order to make it easier to know what it needs to be there.